File: api/src/main/java/com/cloudmen/cloudguard/dto/OrgUnitPolicyDto.java
package com.cloudmen.cloudguard.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class OrgUnitPolicyDto {
    private String key;
    private String title;
    private String description;
    private String status;
    private String statusClass;
    private String baseExplanation;
    private String inheritanceExplanation;
    private boolean inherited;
    private String settingsLinkText;
    private String adminLink;
}

File: frontend/src/app/models/org-unit/OrgUnitPolicyDto.ts
export interface OrgUnitPolicyDto {
    key: string;
    title: string;
    description: string;
    status: string;
    statusClass: string;
    baseExplanation?: string;
    inheritanceExplanation?: string;
    inherited: boolean;
    source: string;
    settingsLinkText?: string;
    adminLink?: string;
    details?: string;
}